src/CalculatingAverages.java

```diff
@@ -0,0 +1,9 @@
+public class CalculatingAverages {
+    public static double calculateAverage(int[] arr) {
+        double total = 0;
+        for (int i = 0; i < arr.length; i++) {
+            total += (double) arr[i]/arr.length;
+        }
+        return total;
```

Comment:
I would probably to the `/arr.length` here (`return total/arr.length`) instead of on line 5, for clarity.  I definitely had to read this a few times to understand what was going on.  :)